### ISSUE TITLE
Add a tool to parser IDs

### ISSUE THREAD

**Issue by jovany-wang** [ray-project/ray#7120]
Mostly, when we debug, we'd like to know the meanings from an id from logs.

It's hard to parse by ourselves manually.

Add a python scripts called `ray parser_id` as a tool to parse the IDs, such as:
```bash
> ray  parser_id  000ebf001045056080001000000
> It's an object id:
       created_by_task: True
       is_put_object: False
       ...
       ...
```

**Comment by jovany-wang** [comment]
@kfstorm  How do you think of this proposal?

**Comment by kfstorm** [comment]
I'm not sure if it's a common use case that worths adding a new command to the `ray` script.
